### DISCUSSION TITLE
Can we move the token groups of PropertyFilter to a different location

### DISCUSSION THREAD

**Discussion by nidhishkrishnan** [cloudscape-design/components#219] [General]
Currently in the selected tokens shows just below the text field (https://cloudscape.design/components/property-filter/?tabId=playground). Is there any way I can show the filtered tokens of PropertyFilter in a different location

**Comment by johannes-weber** [comment]
No, this isn't something that's currently supported by the PropertyFilter component.
Can you please tell us a more about your use case why you want to display it in a different location?
